### DISCUSSION TITLE
Custom labels in LinkField class

### DISCUSSION THREAD

**Discussion by westguard** [UseDataKit/SDK#3] [Ideas]
In the LinkField class, perhaps you could add a method for setting the label of the link. At present it defaults to the value, but sometimes links can be really long, so being able to set the label would improve the UI of the table.

For example:

`LinkField::create( 'pdf_url', __('PDF') )->sortable()->always_visible()->label( __('Download PDF') ),`

**Comment by zackkatz** [comment]
Yes! Good call.

**Comment by doekenorg** [comment]
Added on the `main` branch. There are now 2 methods: `->with_label( string $label )` to overwrite the label of the link. And `->without_label()` to revert this behaviour.
